static/js/components/CommentEntry.jsx

```diff
@@ -74,20 +115,71 @@ const CommentEntry = ({ addComment }) => {
     return formState.group_ids?.filter((value) => Boolean(value)).length >= 1;
   };
 
+  const handleClickSuggestedUsername = (username) => {
```

Comment:
We should also add a keyboard option for selecting the completion.

Reply:
See comment below -- this is already possible